cmd/daemon.go

```diff
@@ -15,17 +15,37 @@
 package cmd
 
 import (
+	"context"
+	"encoding/json"
+	"errors"
 	"fmt"
+	"io/ioutil"
 	"net/http"
+	"os"
+	"strings"
 
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
```

Comment:
Can we rename this? `docker "github.com/docker/docker/client"`. Odd to read things like `client.Client`, no reference to the fact that this is a docker client.